### DISCUSSION TITLE
有个页面不能下载视频啊

### DISCUSSION THREAD

**Discussion by m2905076048** [the1812/Bilibili-Evolved#4073] [综合讨论 / General]
这里打开视频页面咋没有下载视频和弹幕那些功能了 ，是要单独设置吗
![图片](https://user-images.githubusercontent.com/47288778/225517879-3651bc71-a313-41e0-9012-c0eaa365a473.png)
![图片](https://user-images.githubusercontent.com/47288778/225517883-c9426fbe-6c71-4ae5-8658-c383ba02b5c7.png)

**Comment by m2905076048** [comment]
点投稿合集的播放全部后的视频播放页面没有下载了，其他单独的播放页就有

**Reply by m2905076048** [reply]
简介也没有完全展开
![图片](https://user-images.githubusercontent.com/47288778/225518763-c99127ad-b480-4a02-bbea-f15d5a8b9b38.png)

**Reply by Geeyun-JY3** [reply]
请善用搜索，看置顶：#3916
